### PR TITLE
Add warning banner to indicate pre-release

### PR DESCRIPTION
**Pull request by Reflejo**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,6 +15,7 @@
     <title>Envoy Mobile</title>
   </head>
   <body>
+    <div id="banner"><b>IMPORTANT</b> This project is in a pre-release stage, and is not available for production use.</div>
```
Comment (mattklein123) [inline index.html]:
```suggestion
    <div id="banner"><b>IMPORTANT</b> This project is in a pre-release stage. All features are not implemented and it is not ready for production use.</div>
```

Comment (rebello95) [reply index.html]: > All features are not implemented

IMO this reads a little oddly. Maybe `Not all features are implemented` or `No features are implemented`? (Not quite sure what the intent was)

Comment (mattklein123) [reply index.html]: Sure +1 "Not all features are implemented..."

Comment (Reflejo) [reply index.html]:
```
This project is in a pre-release stage. Not all features are implemented and it is not ready for production use.
```
?

Comment (rebello95) [reply index.html]: 🚢

**Pull request merged**